packages/curve-ui-kit/src/widgets/Footer/Footer.tsx

```diff
@@ -0,0 +1,55 @@
+import { styled } from '@mui/material/styles'
+import Grid from '@mui/material/Grid2'
+
+import { SizesAndSpaces } from 'curve-ui-kit/src/themes/design/1_sizes_spaces'
+import { LlamaImg } from 'ui/src/images'
+
+import { Description } from './Description'
+import { Section } from './Section'
+import { sections } from './Sections'
+
+const LlamaImageSrc = (LlamaImg as unknown as { src: string }).src
+const Llama = styled('img')({
+  alt: 'Llama',
+  position: 'absolute',
+})
+
+export const Footer = () => (
+  <Grid
+    container
+    component="footer"
+    data-testid="footer"
+    spacing={1}
+    rowGap={4}
+    sx={(t) => ({
+      position: 'relative',
+      paddingInline: SizesAndSpaces.Spacing.lg,
+      paddingBlock: SizesAndSpaces.Spacing.xl,
+      backgroundColor: t.design.Layer[3].Fill,
+    })}
+  >
+    <Grid
+      size={{
+        mobile: 12,
+        desktop: 3,
+      }}
+    >
+      <Description />
+    </Grid>
+
+    {sections.map((section, i) => (
+      <Grid
+        key={i}
+        size={{
+          mobile: 12,
+          tablet: 4,
+          desktop: 3,
+        }}
+      >
+        <Section key={section.title} {...section} />
```

Comment:
We only need the key in the item generated from the loop. Better to avoid the index if we can, so react can still work well if the order changes (not the case here I guess)

```suggestion
        key={section.title}
        size={{
          mobile: 12,
          tablet: 4,
          desktop: 3,
        }}
      >
        <Section {...section} />
```

Reply:
Should be fixed in https://github.com/curvefi/curve-frontend/pull/500/commits/17db9684935a0e9b4f8a356860f9eb1fdff0dc50